paper.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: Paper and Analysis
+---
+
+# Publications and analysis using CSP.guru
```

Comment:
I guess this should be the title of the page. The title is given in line 3 however. So this line seems unnecessary to me.